src/KOKKOS/sna_kokkos_impl.h

```diff
@@ -624,6 +857,7 @@ void SNAKokkos<DeviceType>::compute_deidrj_cpu(const typename Kokkos::TeamPolicy
 
 /* ----------------------------------------------------------------------
    compute Bi by summing conj(Ui)*Zi
+   not updated yet
```

Comment:
@weinbe2 this is a little vague, can you clarify this comment a little?

Reply:
@stanmoore1 I'll push a fix for that in a moment. tl;dr it can be deleted, it's a casualty of a mistake I made merging in my changes.